main.tex

```diff
@@ -759,7 +758,7 @@
 \item \linkgreen{https://ipfs.io/ipfs/Qmf2rKkDPSsvdudwSmdDPbZuYae8XRV26c1wAFCCvg8Dhw}{Brand-new stack}
 \item \linkgreen{https://ipfs.io/ipfs/QmeS4LjoL1iMNRGuyYSx78RAtubTT2bioSGnsvoaupcHR6}{Search engines information retrieval in practice}
 \item \linkgreen{https://ipfs.io/ipfs/QmNrAFz34SLqkzhSg4wAYYJeokfJU5hBEpkT4hPRi226y9}{Motivating game for adversarial example research}
-\item \linkgreen{https://ipfs.io/ipfs/QmfAEiTpuMjMzTxeRhUW73S7oeJ5za32z1hD3Pi4NBfP7D}{An idea of decentralized search}
+\item \linkgreen{https://ipfs.io/ipfs/QmPnFTJ3m4fHxubXViWyd8dCXor5rjiwYBvF7ws6Vedkmf}{An idea of decentralized search}
```

Comment:
Original link goes go to pure content of this post: https://steemit.com/web3/@hipster/an-idea-of-decentralized-search-for-web3-ce860d61defe5est

Reply:
+